runelite-client/src/test/java/net/runelite/client/plugins/itemstats/ItemStatEffectTest.java

```diff
@@ -93,15 +145,59 @@ public void prepare()
 			.thenReturn(0);
 	}
 
+	private void prepareInventory()
+	{
+		final Item holy_wrench = new Item(ItemID.HOLY_WRENCH, 1);
+		final ItemContainer invi = mock(ItemContainer.class);
+		when(invi.getItems())
+			.thenReturn(new Item[]{holy_wrench});
+
+		when(client.getItemContainer(eq(InventoryID.INVENTORY)))
+			.thenReturn(invi);
+	}
+
 	@Test
 	public void testRevitalisationPlus()
 	{
 		final Effect item = itemStats.get(ItemID.REVITALISATION_1_20957);
-		matchWikiTable(REVITALISATION_TABLE, item);
+		matchWikiTable(REVITALISATION_TABLE, item, false);
 	}
 
-	private void matchWikiTable(final ImmutableMap<Integer, Integer> table, final Effect item)
+	@Test
+	public void testSuperRestore()
 	{
+		final Effect item = itemStats.get(ItemID.SUPER_RESTORE1);
+		matchWikiTable(SUPER_RESTORE_TABLE, item, false);
+	}
+
+	@Test
+	public void testSuperRestoreWithWrench()
+	{
+		prepareInventory();
+
+		final Effect item = itemStats.get(ItemID.SUPER_RESTORE1);
+		matchWikiTable(SUPER_RESTORE_TABLE, item, true);
+	}
+
+	@Test
+	public void testSanfewSerum()
+	{
+		final Effect item = itemStats.get(ItemID.SANFEW_SERUM1);
+		matchWikiTable(SANFEW_TABLE, item, false);
+	}
+
+	@Test
+	public void testSanfewSerumWithWrench()
+	{
+		prepareInventory();
+
+		final Effect item = itemStats.get(ItemID.SANFEW_SERUM1);
+		matchWikiTable(SANFEW_TABLE, item, true);
+	}
+
+	private void matchWikiTable(final ImmutableMap<Integer, Integer> table, final Effect item, boolean hasWrench)
```

Comment:
Given that the wrench test code is exactly the same as the implementation, I don't see a reason to test it.